packages/editor-ui/src/views/SettingsApiView.vue

```diff
@@ -64,7 +64,8 @@
 			</div>
 		</div>
 		<n8n-action-box
-			v-else-if="isTrialing"
+			v-else-if="!isPublicApiEnabled && isTrialing"
```

Comment:
Just checking. Wouldn't this be `|| isTrialing`?

Reply:
We only disable the API for trialing users in cloud. And the CTA specifically mention trialing users, so we only show it then. We do not have CTA for non trialing users, because trialing is the only instance where we block the API. In all other cases, all cloud plans, and community edition, the API is always enabled.

Reply:
Makes sense. Thanks for the explanation 🙏